identity/identity.proto

```diff
@@ -139,15 +139,38 @@ message Capability {
     Type type = 1;
   }
 
+  // VolumeReplication contains the features of the volumereplication operation that the
+  // CSI-driver supports.
+  message VolumeReplication {
+    // Type describes a CSI Service that CSI-drivers can support.
+    enum Type {
+      // UNKNOWN indicates that the CSI-driver does not support the VolumeReplication
+      // operation in the current mode. The CSI-Addons CO plugin will most
+      // likely ignore this node for the VolumeReplication operation.
+      UNKNOWN = 0;
+
+      // VOLUME_REPLICATION indicates that the CSI-driver provides RPCs for a
+      // VolumeReplication operation.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require access to the storage system,
+      // similar to the CSI Controller (provisioner).
+      VOLUME_REPLICATION = 1;
```

Comment:
maybe replication has optional features, like:
- promote/demote: some storage providers might not need it
- resync volume: could be optional too?

Reply:
For now we can go with this to keep it simple and once the migration is done we can decide on adding these options.

Reply:
ok, but note that making something optional after the fact is not as easy as doing it from the start

Reply:
This works for new now to advertise that the csi driver supports Volume_replication. 

We will have two options to discover optional supports 

* we can introduce one more RPC call to check what different capabilities if the volume Replication is supported, the driver can respond what different caps (enable/disable, promote/demote,resync) csi driver supports

* And more fields to the VolumeReplication for individual caps.